SU-SWS/stanford_profile
# D8CORE-5696 Only display location address once

# READY FOR REVIEW

# Summary
- Remove redundant event location settings for the views.

# Need Review By (Date)
- 4/30

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. import configs
3. create an event and populate the location address with any data
4. view the `/events` page
5. verify the location address is only displayed onces.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
